x11-packages/alacritty/freetype-sys.diff

```diff
@@ -1,13 +0,0 @@
-diff --git a/build.rs b/build.rs
-index 4948cf1..363299d 100644
---- a/build.rs
-+++ b/build.rs
-@@ -13,7 +13,7 @@ fn add_sources(build: &mut cc::Build, root: &str, files: &[&str]) {
- 
- fn main() {
-     let target = env::var("TARGET").unwrap();
--    if !target.contains("android")
-+    if true
-         && !target.contains("ohos")
```

Comment:
Why was this patch file deleted? This will make `alacritty` not use the system's `freetype`.